tests/integration/__tests__/__snapshots__/infer-parser.js.snap

```diff
@@ -83,6 +83,14 @@ exports[`Interpreters (stdout) 1`] = `
 
 exports[`Known/Unknown (stdout) 1`] = `"known.js"`;
 
+exports[`isSupported (stdout) 1`] = `
+"{
+  "ignored": false,
+  "inferredParser": "parser-name-inferred-from-language-is-supported"
+}
+"
+`;
```

Comment:
Please rebase, there was a bug in CLI #17332

Reply:
Done!